internal/reconciliations/istio_resources/configmap_control_plane.yaml

```
@@ -0,0 +1,1763 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: istio-control-plane-grafana-dashboard
+  namespace: kyma-system
+  labels:
+    grafana_dashboard: "1"
+    app: monitoring-grafana
+data:
+  istio-workload-dashboard.json: |-
+    {
+      "annotations": {
+        "list": [
+          {
+            "builtIn": 1,
+            "datasource": "-- Grafana --",
+            "enable": true,
+            "hide": true,
+            "iconColor": "rgba(0, 211, 255, 1)",
+            "name": "Annotations & Alerts",
+            "type": "dashboard"
+          }
+        ]
+      },
+      "description": "Istio Control Plane Dashboard version 1.17.1",
```

Honestly, we might just want to remove the version from the dashboards
```suggestion
      "description": "Istio Control Plane Dashboard version 1.18.2",
```

---

Yes, agree, lets remove this version.